core/src/main/scala/net/ruippeixotog/scalascraper/util/LineCompare.scala

```diff
@@ -0,0 +1,22 @@
+package net.ruippeixotog.scalascraper.util
+
+object LineCompare {
```

Comment:
This file should be defined in the test scope, not in the compile scope.

Reply:
Done.